template.yaml

```diff
@@ -102,8 +101,33 @@ Resources:
         openapi: 3.0.3
         info:
           title: NVA Search API
-          version: '0.0.2'
+          version: '0.1.0'
         paths:
+          /:
```

Comment:
Is this actually updated with params?

Reply:
OpenAåpi is updated, but it doesn't show within the current setup - this is for all our api's defined in openapi..